tools/clangrttool/clangrttool.py

```diff
@@ -124,6 +124,10 @@ def run(self):
           "Could not find any clang runtime libraries to package."
           "This is likely a configuration error")
 
+    if "DEVELOPER_DIR" in os.environ:
+      xcode_default_location = "/Applications/Xcode.app/Contents/Developer"
+      clang_lib_path = clang_lib_path.replace(xcode_default_location, os.environ["DEVELOPER_DIR"])
```

Comment:
is this default path always the one here? since it's extracted from the binary's rpaths won't it be just whatever absolute xcode path was used to build the thing?

Reply:
You run into the issue if Xcode resides in different locations locally and on remote builders and the the script here is called in a different environment than when the binary was linked.
Thinking about it again, not all combinations are covered with this fix. Xcode-beta.app locally and Xcode-15.app on the remote builder would still run into the issue, given the actions run in the different environments.

Reply:
i think we could do some regex replace here to be more resilient. at the very least doing `/Applications/Xcode.*.app/Contents/Developer` would cover a lot more cases, but we could probably do something smarter too. Maybe even `.*/Contents/Developer`